self_test/faceted_surface_vtu_output/faceted_surface_vtu_output_tester.cc

```diff
@@ -93,16 +98,22 @@ int main()
   Vector<double> offset(3, 0.0); // Set the rectangle at the origin
   RectangularTetMeshFacetedSurface rectangular_faceted_surface(
     half_x_width, half_y_length, offset, dummy_id);
+  
   // Output into the Paraview .vtu format
   rectangular_faceted_surface.output_paraview("rectangular_faceted.vtu");
 
 
-  // DiskWithTwoLayersTetMeshFacetedSurface
-  //----------------------------------------
-  // Create a disk with two layers, looks like a coin
-  DiskWithTwoLayersTetMeshFacetedSurface disk_with_two_layers_faceted_surface(
-    &disk, half_nsegment, dummy_id, dummy_id, dummy_id, dummy_id, dummy_id);
-  // Output into the Paraview .vtu format
-  disk_with_two_layers_faceted_surface.output_paraview(
-    "disk_with_two_layers_faceted.vtu");
-}
\ No newline at end of file
+  // TAKEN OUT: INVOLVES TRIANGLE AND IS NON-DETERMINISTIC4
+
+//   // DiskWithTwoLayersTetMeshFacetedSurface
+//   //----------------------------------------
+//   // Create a disk with two layers, looks like a coin
+//   DiskWithTwoLayersTetMeshFacetedSurface disk_with_two_layers_faceted_surface(
+//     &disk, half_nsegment, dummy_id, dummy_id, dummy_id, dummy_id, dummy_id);
+
+//   // Output into the Paraview .vtu format
+//   disk_with_two_layers_faceted_surface.output_paraview(
+//     "disk_with_two_layers_faceted.vtu");
```

Comment:
Could we remove the commented-out code? Or add a flag for non-deterministic tests? Or move it to its own script.

Reply:
Done.